lib/devices_test.go

```diff
@@ -281,27 +206,18 @@ func TestAddDeviceLogAndGetLog(t *testing.T) {
 	logtime := time.Now()
 	addDeviceLog(d, logtime)
 	got := GetLog()
-	if len(got) == 0 || got[0].Text != "TestLog" || got[0].State != StateAdded {
-		t.Errorf("got %+v", got)
-	}
+	assert.NotEmpty(t, got, "log should not be empty")
```

Comment:
This is wrong, we want exactly one log line to be created for a single call of `addDeviceLog`